### PR TITLE
Implement [M3C] Hideous Taskmaster

### PR DESCRIPTION
**Pull request by grimreap124**
[[Hideous Taskmaster]]

### CODE REVIEW

File: Mage.Sets/src/mage/cards/h/HideousTaskmaster.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import mage.constants.Duration;
+import mage.target.TargetPermanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.targetadjustment.EachOpponentPermanentTargetsAdjuster;
+import mage.target.targetpointer.EachTargetPointer;
+
+
+/**
+ *
+ * @author anonymous
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/h/HideousTaskmaster.java:27]: You can set authorship if you desire so. 

Btw you can set up a author file in the Utils folder to automatically set authorship of generated files.

File: Mage.Sets/src/mage/cards/h/HideousTaskmaster.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        this.subtype.add(SubType.ELDRAZI);
+        this.power = new MageInt(7);
+        this.toughness = new MageInt(2);
+
+        // Devoid
+        this.addAbility(new DevoidAbility(this.color));
+
+        // When you cast this spell, for each opponent, gain control of up to one target creature that player controls until end of turn.
+        // Untap those creatures. They gain trample, haste, and annihilator 1 until end of turn.
+        Ability ability = new CastSourceTriggeredAbility(new GainControlTargetEffect(Duration.EndOfTurn).setTargetPointer(new EachTargetPointer()));
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/h/HideousTaskmaster.java:43]: I expect this needs a lot of text adjustment (with setText on effects). Can be done later if you don't want to do it here.

Comment (grimreap124) [reply Mage.Sets/src/mage/cards/h/HideousTaskmaster.java:43]: I'll have a look at it tomorrow. Thanks for pointing it out, I haven't implemented many cards

**Review by Susucre — APPROVED** [review]
Still some text issues, but don't worry I'll fix them after

**Comment by Susucre** [comment]
Ok, I entirely blank on the fact that Taskmaster also has Trample Haste and Annihilator 1.
Adding those in a few minutes on master.

**Pull request merged**